packages/concerto-core/test/serializer/objectvalidator.js

```diff
@@ -264,7 +264,7 @@ describe('ObjectValidator', function () {
             parameters.stack = new TypedStack(data);
             (function () {
                 objectValidator.visit(concerto.getTypeDeclaration(data), parameters);
-            }).should.throw(/expected a Relationship/);
+            }).should.throw('Model violation in the "undefined" instance. Class "test.Person" has a value of "[object Object]". Expected a "Relationship".');
```

Comment:
I'm wondering if the `undefined` here is a bug? the sentence doesn't read quite right. Could we double check?